platforms/nuttx/cmake/init.cmake

```diff
@@ -77,6 +77,10 @@ execute_process(
 	OUTPUT_FILE ${CMAKE_CURRENT_BINARY_DIR}/nuttx_olddefconfig.log
 	RESULT_VARIABLE ret
 )
+execute_process(
+	COMMAND ${CMAKE_COMMAND} -E touch ${CMAKE_CURRENT_BINARY_DIR}/defconfig_inflate_stamp
+	WORKING_DIRECTORY ${NUTTX_DIR}
+)
```

Comment:
It probably doesn't matter for certain simple standalone cases, but I wonder if it should be done here as well for consistency.

Reply:
I can add it if you prefer, it just gets a bit more complex.